README.md

```diff
@@ -4,9 +4,80 @@
 
 Run Wharf locally on your own machine using [Docker Compose](https://docs.docker.com/compose/)
 
-For documentation on how to set this up, please visit
+```console
+$ docker-compose -f docker-compose.yml pull
```

Comment:
Question is if we should rename `docker-compose.override.yml` to something like `docker-compose.dev.yml` so it doesn't get loaded by default by Docker Compose. And then when you link the files according to the documentation, you do:

```console
$ cd ..

$ ls -1
wharf-api
wharf-docker-compose
wharf-provider-azuredevops
wharf-provider-github
wharf-provider-gitlab
wharf-web

$ ln -sfv wharf-docker-compose/docker-compose.yml docker-compose.yml
'docker-compose.yml' -> 'wharf-docker-compose/docker-compose.yml'

$ ln -sfv wharf-docker-compose/docker-compose.dev.yml docker-compose.override.yml
'docker-compose.override.yml' -> 'wharf-docker-compose/docker-compose.override.yml'
```

Where the last one there links `docker-compose.override.yml` to `wharf-docker-compose/docker-compose.dev.yml`

Thoughts?

Reply:
Talked myself into it. I've updated the docs and files